src/cli/service/echo.rs

```diff
@@ -416,7 +416,11 @@ impl Service<(), Request> for EchoService {
                         }),
                         ClientHelloExtension::Opaque { id, data } => json!({
                             "id": id.to_string(),
-                            "data": format!("0x{}", hex::encode(data)),
+                            "data": if data.is_empty() {
```

Comment:
Same here, I would only return the "data" property if we have data, and otherwise just only have a object with the single "id" property in there